src/sandstorm/run-bundle.c++

```diff
@@ -1689,128 +1511,12 @@ private:
   }
 
   Config readConfig() {
-    // Read and return the config file.
-    //
-    // If parseUids is true, we initialize `uids` from SERVER_USER.  This requires shelling
```

Comment:
So the code had a comment mentioning `parseUids`, but no such option actually existed?

But you've added this option in the factored-out code?

Reply:
Correct.

Reply:
Does the UID parsing not work in the `spk dev` use case, or did you only skip it because it's not needed?